starter-code/src/clue.js

```diff
@@ -92,5 +92,23 @@ const weaponsArray = [
 ];
 
 // ITERATION 2
+function selectRandom(arr){
+ return arr[Math.floor(Math.random() * arr.length)];
 
+}
+function pickMystery(){
+    
+    
+    var x={
+        suspect :selectRandom.suspectsArray,
```

Comment:
This returns undefined. To get a random item it should have been something like suspect:selectRandom(suspectsArray)

Reply:
I tried this and it was still undefined ....